libs/core/src/components/sage-tab/sage-tab.tsx

```diff
@@ -0,0 +1,83 @@
+import { Component, Element, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+@Component({
+  tag: 'sage-tab',
+  styleUrl: 'sage-tab.scss',
+  shadow: false,
+})
+export class SageTab {
+  @Element() el: HTMLElement;
+
+  /**
+   * Sets the related tab name, this name must match a `sage-tabpanel`'s tab property, required
+  */
+  @Prop() tab: string;
+
+  /**
+   * Keeps track of the activeTab, this property is passed in by parent component
+  */
+  @Prop({mutable: true}) activeTab: string;
```

Comment:
Why the use of mutable?

Reply:
After reading through the stencil docs, I've found that a mutable prop allows stencil to know a prop will change over time without having to use a `@State` decorator which, from what I've ready, its recommend only for use when absolutely necessary. Because of this, I've opted for a mutable prop over state.